hpx/runtime/threads/executors/limiting_executor.hpp

```diff
@@ -1,207 +1,16 @@
-//  Copyright (c) 2017-2018 John Biddiscombe
+//  Copyright (c) 2017-2019 John Biddiscombe
 //
 //  SPDX-License-Identifier: BSL-1.0
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-#ifndef HPX_RUNTIME_THREADS_LIMITING_EXECUTOR_HPP
-#define HPX_RUNTIME_THREADS_LIMITING_EXECUTOR_HPP
+#ifndef HPX_RUNTIME_THREADS_LIMITING_EXECUTOR
```

Comment:
Missing `_HPP`?

Reply:
Good catch on all of these, thanks! Most likely caused by my sloppy find-and-replacing.